_people/JayeNias

```diff
@@ -0,0 +1,12 @@
+---
+first_name: Jaye
+last_name: Nias
+title: Assistant Professor of Computer and Information Sciences
+school: Spelman College
+website: https://www.jayenias.com/
+image: https://static.wixstatic.com/media/4a7add_2ea535ea9ee74b3e8adfd1324bbec06e~mv2.jpg/v1/fill/w_468,h_603,al_c,q_80,usm_0.66_1.00_0.01/Pushing%2520through%2520the%2520challenges%2520_.webp
+---
+Prof. Nias is a researcher in computational social science in Spelman College's Course-Based Undergraduate Research Experiences (CURE) Program. Her current research focus is on developing educational computational technologies to promote cultural diversity and societal inclusion. She has also conducted research in human computer interaction, with a focus on child-computer interactions and mobile computing.
+* [Nias, J.and Ruffin, M. CultureBot: A Culturally Relevant Humanoid Robot Dialogue Agent. In 2020 ACM Southeast Conference (ACMSE 2020), April 2–4, 2020, Tampa, FL, USA. ACM, New York, NY, USA.](https://dl.acm.org/doi/10.1145/3374135.3385306)
```

Comment:
Check formatting from other citations, but we've been linking just the article title and not the whole citation. If the file extension is .md there will be a preview button on github.